File: mobile/lib/profile/domain/gender.dart
/// Mirrors `App\Enums\Gender` on the backend — confirmed set, open decision
/// #6 (docs/05-open-decisions.md). Don't add a fourth value without that
/// decision changing (see CLAUDE.md "handling open decisions").
enum Gender {
  man('man', 'Man'),
  woman('woman', 'Woman'),
  nonBinary('non_binary', 'Non-binary');

  const Gender(this.apiValue, this.label);

  final String apiValue;
  final String label;

  static Gender fromApiValue(String value) =>
      Gender.values.firstWhere((g) => g.apiValue == value);
}

File: backend/app/Enums/Gender.php
<?php

namespace App\Enums;

/**
 * profiles.gender and user_preferences.interested_in_genders values.
 * Confirmed set — open decision #6 (docs/05-open-decisions.md). Don't add a
 * fourth option without that decision changing; see CLAUDE.md "handling open
 * decisions".
 */
enum Gender: string
{
    case Man = 'man';
    case Woman = 'woman';
    case NonBinary = 'non_binary';
}
